ntc_templates/templates/cisco_asa_show_route.textfsm

```diff
@@ -35,6 +35,13 @@ ROUTES
   ^${PROTOCOL}(\*){0,1}\s${TYPE}\s+${NETWORK}\s+${NETMASK}\sis\sdirectly\sconnected,\s${NEXTHOPIF} -> Record
   ^\s+is\sdirectly\sconnected,\s${NEXTHOPIF} -> Record
   #
+  # Match VPN routes
+  ^${PROTOCOL}\s${TYPE}\s+${NETWORK}\s${NETMASK}\sconnected\sby\sVPN\s\(advertised\),\s${NEXTHOPIF}\s*$$ -> Record
+  #
+  # Match multiline VPN routes
+  ^${PROTOCOL}\s${TYPE}\s+${NETWORK}\s${NETMASK}\s*$$ -> Continue
+  ^\s+connected\sby\sVPN\s\(advertised\),\s${NEXTHOPIF}\s*$$ -> Record
```

Comment:
Making the whitespace matching a bit more robust for the long-term.
_(yeah, quite minor)_

```suggestion
  # Match VPN routes
  ^${PROTOCOL}\s+${TYPE}\s+${NETWORK}\s+${NETMASK}\s+connected\s+by\s+VPN\s+\(advertised\),\s+${NEXTHOPIF}\s*$$ -> Record
  #
  # Match multiline VPN routes
  ^${PROTOCOL}\s+${TYPE}\s+${NETWORK}\s+${NETMASK}\s*$$ -> Continue
  ^\s+connected\s+by\s+VPN\s+\(advertised\),\s+${NEXTHOPIF}\s*$$ -> Record
```